config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -17,7 +17,7 @@ presets:
   - name: NON_MASQUERADE_CIDR
     value: 0.0.0.0/0
   - name: KUBELET_TEST_ARGS
-    value: --runtime-cgroups=/system.slice/containerd.service
+    value: --runtime-cgroups=/system.slice/containerd.service --cgroup-driver=systemd
```

Comment:
I think it would still require a comma
`--runtime-cgroups=/system.slice/containerd.service,--cgroup-driver=systemd`

Reply:
Arent these flags passed directly to the kubelet?

Reply:
yep right, it should work the way you specified